tools/buildPackage.js

```diff
@@ -154,10 +162,27 @@ if (isLinux) {
   cmds.push('copy .\\res\\start-tile-150.png "' + path.join(buildDir, 'resources', 'start-tile-150.png') + '"')
   cmds.push('makensis.exe -DARCH=' + arch + ` res/${channel}/braveDefaults.nsi`)
   cmds.push('ncp ./app/extensions ' + path.join(buildDir, 'resources', 'extensions'))
+
   // Make sure the Brave.exe binary is squirrel aware so we get squirrel events and so that Squirrel doesn't auto create shortcuts.
   cmds.push(`"node_modules/rcedit/bin/rcedit.exe" ./${appName}-win32-` + arch + `/${appName}.exe --set-version-string "SquirrelAwareVersion" "1"`)
 }
 
+// Verify tor binaries and bundle with Brave
+var torPath
+if (isDarwin) {
+  torPath = path.join(buildDir, `${appName}.app`, 'Contents', 'Resources', 'extensions', 'bin')
+} else {
+  torPath = path.join(buildDir, 'resources', 'extensions', 'bin')
+}
+cmds.push('mkdirp ' + torPath)
+cmds.push('curl -o ' + path.join(torPath, 'tor') + ' ' + torURL)
+cmds.push('curl -o ' + path.join(torPath, 'tor-sig') + ' ' + torSigURL)
+cmds.push('gpg --verify ' + path.join(torPath, 'tor-sig') + ' ' + path.join(torPath, 'tor'))
```

Comment:
do you have to import the key first? my gpg shows `Can't check signature: No public key`

is this key (`D3B76046641D2C275EF8C728993DAAF0298ED4FE`) owned by brave or tor? if by us, who has control of it?

Reply:
Its a demo key that I generated. We can switch the key and updated the `signature` files once we are ready to ship.

```
-----BEGIN PGP PUBLIC KEY BLOCK-----

mQINBFqrIn4BEADgqJTNhqw8B/T7huOm31dDzkqJEB8pIpcML35POcYs5CwOvGO2
s2rzCh6Atys212CCZEMs4spWAReXux0KirKq0ON9N8modct2AcNS5bSdmyMhQehP
wb1jt3bVX6nufJqrlYIf34LdHx3DAUhLr8bFSK9Hskt15Q5UyT0UKUfSl5t69zLf
DFZObR0aHqltfuenFfquO6aUpXopyhd6L89Jsm/o7vME6vdG/TJ3U97AV5wwDlrH
mRUWrXxgqcPrrKPJ0SdQ3cf36iPdvglsVC0m9LU6XzTph61gD1JnXmYmxmRtubYu
wr9N2JB+aTxR3TsOn3hfpWVrLGME6wpFMkvRzVaHQc8PUJhVMmeZHXECr264/R9d
PDCAFoqaAiXjhvDbtYK2iWTdos2WWFZ+SdfSQaNBv37nQ5UO2sZfa/iKwVzu7Xka
VHCKckm37PBNQwVmwUjunTa3TpKE7Mxygm1v3dCofxCm+Eq9/4uyL6sVMMbtBiUI
1rY9G8zyF8Z+n+jQmiMHtwk2pBp7lQf4bnSUrc401MQ2WOX67COU2DY4j4cUaGzi
6/StRDUKX+3InXEaNymPYuelAK6iafh+TvOeH4nRfz/mmlL4deWZKeyuEgqZK6E3
uCxyUbwXwfqbs0REma58gl7qCOOmBRDUxGlOryxdHGph4Hj4MK9GcDazEwARAQAB
tC5QcmFuamFsIEp1bWRlIChEZW1vKSA8cHJhbmphbC5qdW1kZUBnbWFpbC5jb20+
iQJOBBMBCAA4FiEE07dgRmQdLCde+McomT2q8CmO1P4FAlqrIn4CGwMFCwkIBwIG
FQoJCAsCBBYCAwECHgECF4AACgkQmT2q8CmO1P7PkQ/+KLBGeErNQTV0I6dvSTdd
KfQTcLmc1xnGIqcCYP1evu54XmZmHJwr6O7TdRovo/ycotKeKbZLug6QTr9Az+uv
Gd1hL3WC4D+67KDbJHupCowW3RPKYmhHTgvKC74czN5IzAX8CsjtXMSWbCS9iqgG
QewQJsfHvvFG+r1HwUs37N7L0Oev1g2qzdP+pLOKwAgA62o7xUtUIQETUPjO1D9P
G09G2QPLGwEQwPOaSk2dom5bmHBBVkAXXrWmgZ8cDdIJJxGcFneRcIYBhA6wxAGz
vOKnTWYN6W4OC4X4TDUhrUTdwCjoScBdrX4GAbolWeI++d23b/6PL47HtnGiTl64
KtvmKKSqSvImOHBT78GmI0mdZ9VBfbyr5blC/wuww/Du3ahVvdsFv+30HEpsOaUd
CyK7A3wiT0O+69M7SpTMhA2/QU3AHZfPDo9EGCAgh9xeljN2Thcqxl+vyAWBteVv
5PUXs46rHU3F1dqHLlGlHTZwY6MOcTSPegHIGV+vZoJwfuj8IQbw/x0712SUJzvX
c4HsBsePkJe3TaRVqVdAysCRK3a/pVDp9Rcf2FCRTXdDKBZeli28/0md81CiGFHf
EaoW/JFFy6d91iW+RB2E8GefYDpa9+JrheUSDV3aPoWprX3yQiIeWz5jpINsQStm
akWrUSRBaq7kcZeMJnJRysA=
=GCo9
-----END PGP PUBLIC KEY BLOCK-----
```

Reply:
can you add the command here to import it for now?

also, are curl and gpg already installed on windows?

Reply:
1. Save the key to the file say `demo-key.asc` 
2. gpg --import demo-key.asc

curl was installed, I updated the windows box with `gpg4win`

Reply:
i meant could you add the import command into this script (or into the setup README) so that people who are building local packages won't get the error :)